kubernetes/structure_persistent_volume_claim.go

```diff
@@ -4,8 +4,7 @@ import (
 	"errors"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-	"k8s.io/api/core/v1"
-	corev1 "k8s.io/api/core/v1"
```

Comment:
What's the reasoning for renaming this import alias? 

The new value of `v1` is superfluous because it's also implied by the last element in the import path. That aside, we tend to rename import paths that end with `v1` to avoid conflicts, often with `k8s.io/apimachinery/pkg/apis/meta/v1`. That isn't yet the case here, but should that need to be imported, we're going to have to rename this yet again, so I'd rather keep the `corev1` alias as is.

Reply:
This was done by a golang auto formatter. We can easily revert it.

Reply:
Reverted